test/acl.test.js

```diff
@@ -90,6 +90,42 @@ describe('security scopes', function() {
 });
 
 describe('security ACLs', function() {
+  it('supports checkPermission() returning a promise', function(done) {
+    ACL.create({
```

Comment:
See http://loopback.io/doc/en/contrib/style-guide.html for more info on promise style tests in mocha